src/networks.ts

```diff
@@ -38,3 +38,38 @@ export const CHAIN_IDs = {
   ...MAINNET_CHAIN_IDs,
   ...TESTNET_CHAIN_IDs,
 };
+
+interface PublicNetwork {
+  name: string;
+  nativeToken: string;
+  blockExplorer: string;
+}
+
+export const PRODUCTION_NETWORKS: { [chainId: number]: PublicNetwork } = {
+  [CHAIN_IDs.ARBITRUM]: { name: "Arbitrum One", nativeToken: "ETH", blockExplorer: "https://arbiscan.io" },
+  [CHAIN_IDs.BASE]: { name: "Base", nativeToken: "ETH", blockExplorer: "https://basescan.org" },
+  [CHAIN_IDs.BOBA]: { name: "Boba", nativeToken: "ETH", blockExplorer: "https://blockexplorer.boba.network" },
+  [CHAIN_IDs.MAINNET]: { name: "Mainnet",  nativeToken: "ETH", blockExplorer: "https://etherscan.io" },
+  [CHAIN_IDs.OPTIMISM]: { name: "Optimism", nativeToken: "ETH", blockExplorer: "https://optimistic.ethercan.io" },
+  [CHAIN_IDs.POLYGON]: { name: "Polygon",  nativeToken: "MATIC", blockExplorer: "https://polygonscan.com" },
+  [CHAIN_IDs.ZK_SYNC]: { name: "zksync", nativeToken: "ETH", blockExplorer: "https://explorer.zksync.io" },
+};
+
+export const TEST_NETWORKS: { [chainId: number]: PublicNetwork } = {
```

Comment:
nb. A few entries have been culled from the upstream UMA defs. This is mostly old testnets that are no longer used, and we'd _never_ have any need for them.